src/components/FormFieldError/FormFieldError.tsx

```diff
@@ -0,0 +1,17 @@
+import { FormHelperText } from '@mui/material';
+import { FieldError } from 'react-hook-form';
+
+type Props = {
+  error: FieldError | undefined;
+};
+
+const FormFieldError = ({ error }: Props) => {
+  if (!error) return <></>;
```

Comment:
```suggestion
  if (!error) return null;
```
[Nit] Null can be used to prevent rendering.